components/SendMoneyToCollectiveBtn.js

```diff
@@ -117,22 +128,39 @@ const addPaymentMethodsData = graphql(paymentMethodsQuery, {
   },
 });
 
-const sendMoneyToCollectiveMutation = gql`
-  mutation SendMoneyToCollective($order: OrderInputType!) {
-    createOrder(order: $order) {
+const collectiveBalanceQuery = gql`
+  query CollectivePage($slug: String!) {
+    Collective(slug: $slug) {
       id
-      fromCollective {
+      stats {
+        balance
+      }
+    }
+  }
+`;
+
+const sendMoneyToCollectiveMutation = gqlV2/* GraphQL */ `
+  mutation SendMoneyToCollective($order: OrderCreateInput!) {
+    createOrder(order: $order) {
+      order {
         id
-        stats {
+        fromAccount {
           id
-          balance
+          stats {
+            balance {
+              currency
+              valueInCents
+            }
+          }
         }
-      }
-      collective {
-        id
-        stats {
+        toAccount {
           id
-          balance
+          stats {
```

Comment:
Is that needed actually?

Reply:
That's true. I removed these, just didn't realize we actually doesn't use them with the v2 api now. 🙂